src/test/java/org/opensearch/knn/ODFERestTestCase.java

```diff
@@ -138,7 +140,7 @@ protected void wipeAllODFEIndices() throws IOException {
 
             for (Map<String, Object> index : parserList) {
                 String indexName = (String) index.get("index");
-                if (indexName != null && !".opendistro_security".equals(indexName)) {
+                if (indexName != null && !".opendistro_security".equals(indexName) && !indexName.matches(KNN_BWC_PREFIX+"(.*)")) {
```

Comment:
Can you create a new method and move criteria to skip deletion separately

Reply:
Updated